source/_drafts/videojs-vr-now-under-the-video-js-org.md

```diff
@@ -0,0 +1,56 @@
+---
+title: videojs-vr now under the video.js org
+author:
+  name: Brandon Casey
+  github: brandonocasey
+tags: vr, 360, plugins
+---
+
+We now support a VR/360 video plugin Video.js 6!
+
+## Where did it come from?
```

Comment:
probably should mention the join effort sponsorship.

Reply:
Would appreciate both mentions and links to StreamShark.io, Brightcove and HapYak who have sponsored the work financially and in kind for the last ~ 12 months.

Reply:
& that StreamShark.io will continue to support and contribute the plugin under the 'official' VJS project.